lib/packwerk/reference_extractor.rb

```diff
@@ -51,20 +51,17 @@ def reference_from_node(node, ancestors:, absolute_file:)
 
     sig do
       params(
-        unresolved_references_and_offenses: T::Array[T.any(UnresolvedReference, Offense)],
+        unresolved_references: T::Array[UnresolvedReference],
         context_provider: ConstantDiscovery
-      ).returns(T::Array[T.any(Reference, Offense)])
+      ).returns(T::Array[Reference])
     end
-    def self.get_fully_qualified_references_and_offenses_from(unresolved_references_and_offenses, context_provider)
-      fully_qualified_references_and_offenses = T.let([], T::Array[T.any(Reference, Offense)])
-
-      unresolved_references_and_offenses.each do |unresolved_references_or_offense|
-        if unresolved_references_or_offense.is_a?(Offense)
-          fully_qualified_references_and_offenses << unresolved_references_or_offense
-
-          next
-        end
+    def self.get_fully_qualified_references_from(
+      unresolved_references,
```

Comment:
Let's keep the arguments in the same line since. the line size isn't > 120 yet.

Reply:
Changed!